webknossos/webknossos/dataset/_array.py

```diff
@@ -593,7 +593,11 @@ def create(cls, path: Path, array_info: ArrayInfo) -> "ZarritaArray":
             ArrayV2.create(
                 store=path,
                 shape=(array_info.shape),
-                chunks=(array_info.num_channels,) + array_info.chunk_shape.to_tuple(),
+                chunks=(array_info.num_channels,)
+                + tuple(
```

Comment:
Please add a comment what is happening here.

Reply:
Done.